Paraswap-Agents/Role-Changes/README.md

```diff
@@ -0,0 +1,45 @@
+# Role Change Detection Agent
+
+## Description
+
+This agent detects role changes in `AccessControl` contract.
+
+## Supported Chains
+
+- Ethereum
+- Polygon
+- BSC
```

Comment:
If all other chains are being referred to with their full name it's probably best to do the same for Binance Smart Chain too.